f8a_worker/workers/githubstats.py

```diff
@@ -0,0 +1,55 @@
+from bs4 import BeautifulSoup
+import requests
+import logging
+from f8a_worker.base import BaseTask
+
+logger = logging.getLogger(__name__)
+
+class GithubStats(BaseTask):
+    """ Collects statistics by parsing Github Page """
+    _analysis_name = "github_manifest_details"
+
+    @classmethod
+    def _get_page(cls, repo_name):
+        github_url = 'https://github.com/'
+        url = github_url + "/" + repo_name
+        raw_data = requests.get(url)
+        if raw_data:
+            soup = BeautifulSoup(raw_data.text, "html.parser")
+            return soup
+        else:
+            logger.info("Could not fetch the page for repo_name:"% repo_name)
```

Comment:
The proper way to handle this would be to raise `FatalTaskError` that gets available after:
```python
from selinon import FatalTaskError
```

If you raise this error system will automatically keep track of failed task (and tasks will not be retried - overrides `retry` configuration option). We have capabilities to retry such failed tasks in future.